beacon-chain/sync/checkpoint/initialize.go

```diff
@@ -0,0 +1,29 @@
+package checkpoint
```

Comment:
Elsewhere I have called this feature "origin sync" so I should probably make this naming more consistent...